hack/release.sh

```diff
@@ -22,6 +22,8 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+DO_WINDOWS_BUILD=${DO_WINDOWS_BUILD_ENV:-true}
```

Comment:
IMO since we build for Linux most of the time it would be nice to keep the default option as false, and explicitly set the env variable to true when we want to build Windows images.

Reply:
we can not do that, as on Github when we make a release or push a new commit, we want to generate windows images. 

we can use this option while developing to speed up creating images.